main_class.js

```diff
@@ -0,0 +1,76 @@
+// Book Div and Template const
+// eslint-disable-next-line max-classes-per-file
+const temp = document.querySelector('.book');
+const bookshelf = document.querySelector('#bookshelf');
+
+const inputTitle = document.querySelector('#title');
+const inputAuthor = document.querySelector('#author');
+
+class Book {
+  constructor(id, title, author) {
+    this.id = id;
+    this.title = title;
+    this.author = author;
+  }
+}
+
+class Library {
+  constructor() {
+    this.library = [];
+    this.startLibrary();
+  }
+
+  startLibrary() {
+    this.reloadLibrary();
+    this.idBook = this.library.length;
+  }
+
+  addBook() {
+    const title = inputTitle.value;
+    const author = inputAuthor.value;
+    this.idBook += 1;
+
+    const book = new Book(this.idBook, title, author);
+    this.library.push(book);
+
+    localStorage.library = JSON.stringify(this.library);
+
+    this.reloadLibrary();
+
+    inputTitle.value = '';
+    inputAuthor.value = '';
+  }
```

Comment:
* Currently when i try to add a book i get this error in the console kindly fix it 🙏 
![Screenshot (149)](https://user-images.githubusercontent.com/52497246/124675512-b6f63380-deb4-11eb-929d-d6f9ef705280.png)

Reply:
Hello! Did you type by accident on the main_class.js? It seems that yours contain an `u` somewhere!